src/java/org/apache/cassandra/index/sai/disk/io/IndexFileUtils.java

```diff
@@ -44,6 +46,7 @@ public class IndexFileUtils
                                                                                              .build();
 
     public static final IndexFileUtils instance = new IndexFileUtils(DEFAULT_WRITER_OPTION);
+    public static final Supplier<Checksum> CHECKSUM_CRC32C_FACTORY = CRC32C::new;
```

Comment:
nit: If we make the other changes suggested, this could probably just become `CHECKSUM_FACTORY`, as it might not always and forever be `CRC32C`. If we did change it, I guess there would be some versioning involved, but no need to think about that now.

Reply:
Renamed to `CHECKSUM_FACTORY`.